Sources/Mailgun/Models/MailgunEmail.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  MailgunEmail.swift
+//  Mailgun
+//
+//  Created by Anthony Castelli on 11/14/16.
+//
+//
+
+import Foundation
+import SMTP
+
+open class MailgunEmail {
+    
+    open var from: EmailAddressRepresentable?
+    open var to: [EmailAddress]?
+    open var subject: String?
+    open var body: EmailBody?
+    
+    public init(to: [EmailAddress], from: EmailAddressRepresentable, subject: String?, body: EmailBody?) {
+        self.to = to
+        self.from = from
+        self.subject = subject
+        self.body = body
+    }
+    
+    internal func dictionaryRepresentation() -> [String : String] {
```

Comment:
In `SendGridEmail`, I have made it conform to `NodeRepresentable` and serialised it for sending that way. I'm happy for you to stay with this method if you really want to, but going with Node seemed more in the Vapor style to me.